src/Controls/src/Core/HandlerImpl/Layout/Layout.Windows.cs

```diff
@@ -16,8 +16,8 @@ public partial class Layout
 
 		static void UpdateInputTransparent(IViewHandler handler, IView layout)
 		{
-			if (handler.PlatformView is FrameworkElement fe)
-				fe.UpdateInputTransparent(handler, layout);
+			if (handler.PlatformView is LayoutPanel lp && handler is ILayoutHandler lh && layout is Maui.ILayout iLayout)
```

Comment:
why only in these cases? can we add a comment?

Reply:
Updated the code